typescript-json-schema.ts

```diff
@@ -1828,6 +1880,70 @@ function normalizeFileName(fn: string): string {
     return fn;
 }
 
+async function lintSchema(schemaJson: string, options: { fix?: boolean; strict?: boolean } = {}): Promise<string> {
+    const tempDir = fs.mkdtempSync(path.join(os.tmpdir(), "jsonschema-lint-"));
+    const tempFile = path.join(tempDir, "schema.json");
+
+    try {
+        fs.writeFileSync(tempFile, schemaJson);
+
+        const jsonschemaCmd = path.join(process.cwd(), "node_modules/.bin/jsonschema");
+        let cmd = `"${jsonschemaCmd}" lint "${tempFile}"`;
+
+        if (options.fix) {
+            cmd += " --fix";
+        }
+
+        if (options.strict) {
+            cmd += " --strict";
+        }
+
+        try {
+            const result = execSync(cmd, {
```

Comment:
I'm not too happy about calling a tool like this. Don't you have a more direct/js way to call your library?